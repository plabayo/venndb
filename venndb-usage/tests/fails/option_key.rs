use venndb::VennDB;

#[derive(Debug, VennDB)]
struct Employee {
    #[venndb(key)]
    id: Option<u32>,
}

fn main() {}
