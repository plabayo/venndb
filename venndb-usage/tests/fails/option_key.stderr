error: Invalid field-level `venndb` attribute
       `key` fields cannot be `Option`
 --> tests/fails/option_key.rs:6:9
  |
6 |     id: Option<u32>,
  |         ^^^^^^
